gson/src/main/java/com/google/gson/internal/Excluder.java

```diff
@@ -240,19 +240,15 @@ private boolean isValidVersion(Since since, Until until) {
   private boolean isValidSince(Since annotation) {
     if (annotation != null) {
       double annotationVersion = annotation.value();
-      if (annotationVersion > version) {
-        return false;
-      }
+      return version >= annotationVersion;
     }
     return true;
   }
 
   private boolean isValidUntil(Until annotation) {
     if (annotation != null) {
       double annotationVersion = annotation.value();
-      if (annotationVersion <= version) {
-        return false;
-      }
+      return version < annotationVersion;
```

Comment:
This should behave equivalently (unless someone uses `NaN` as value...), but is hopefully a bit easier to understand.